src/tbp/monty/frameworks/actions/actions.py

```diff
@@ -99,17 +128,20 @@ def __iter__(self):
                 continue
             yield key, value
 
-    @abstractmethod
-    def act(self, actuator: Actuator) -> None:
-        """Execute the action using the provided actuator."""
-        pass
+
+class LookDownActionSampler(Protocol):
+    def sample_look_down(self, agent_id: str) -> LookDown: ...
```

Comment:
I'm sorry if this is a stupid question but where do we actually define what happens when sample_look_down is called?